qa/rpc-tests/wallet_shieldingcoinbase.py

```diff
@@ -86,17 +86,19 @@ def run_test (self):
             myopid = self.nodes[3].z_sendmany(mytaddr, recipients)
         except JSONRPCException as e:
             errorString = e.error['message']
-        assert_equal("Invalid from address: does not belong to this node, spending key not found.", errorString);
+        assert_equal("Invalid from address, no spending key found for address", errorString);
 
         # This send will fail because our wallet does not allow any change when shielding a coinbase utxo,
         # as it's currently not possible to specify a change address in z_sendmany.
```

Comment:
It's not that the wallet doesn't allow change; it's a consequence of this consensus rule:
> A transaction with one or more transparent inputs from coinbase transactions MUST have no transparent
outputs (i.e. `tx_out_count` MUST be 0). Inputs from coinbase transactions include Founders’ Reward outputs
and funding stream outputs.

Since the input is transparent, the change also can't be transparent, because that would be a transparent output. If we were to specify a shielded address for "change", that would contradict the definition of "change" as funds that are sent to an address that is also spent from in the same transaction.

(Technically, there could be another shielded input that is also where change goes to, but that's not why *this* send will fail, even if the wallet were to support that, because it has no such shielded input.)

Reply:
This wasn't my comment, obviously, but I've fixed it and added a note that this test is expected to start failing after UA integration (which will send the change to the most recent shielded pool corresponding to the account corresponding to the "from" address.)